### PR TITLE
added pv command in message click

### PR DESCRIPTION
**Pull request by efefury**
in pf join message :t-rex: :t-rex: :t-rex: :t-rex: :t-rex: :t-rex: :t-rex: :t-rex: :t-rex: :t-rex:

### CODE REVIEW

File: src/main/java/io/github/moulberry/notenoughupdates/options/seperateSections/Dungeons.java [hunk 1]
```diff
@@ -272,4 +272,13 @@ public class Dungeons {
 	@ConfigAccordionId(id = 2)
 	public String dungBatColour = "0:255:12:255:0";
 
+	@Expose
+	@ConfigOption(
+		name = "Open Pv On Join ",
```
Comment (jani270) [inline src/main/java/io/github/moulberry/notenoughupdates/options/seperateSections/Dungeons.java:277]: Please remove the space at the end

**Review by jani270 — CHANGES_REQUESTED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/options/seperateSections/Dungeons.java (hunk 1 above)
Comment (jani270) [reply src/main/java/io/github/moulberry/notenoughupdates/options/seperateSections/Dungeons.java:277]: and please name it better

**Comment by Ascynx** [comment]
:watching:

**Review by NopoTheGamer — APPROVED** [review]

File: src/main/java/io/github/moulberry/notenoughupdates/options/seperateSections/Dungeons.java [hunk 2]
```diff
@@ -272,4 +272,13 @@ public class Dungeons {
 	@ConfigAccordionId(id = 2)
 	public String dungBatColour = "0:255:12:255:0";
 
+	@Expose
+	@ConfigOption(
+		name = "Add Pv Command Click on Party Finder Join",
```
Comment (hannibal002) [inline src/main/java/io/github/moulberry/notenoughupdates/options/seperateSections/Dungeons.java:277]: Need to shorten this name

Comment (hannibal002) [reply src/main/java/io/github/moulberry/notenoughupdates/options/seperateSections/Dungeons.java:277]: image

**Review by NopoTheGamer — DISMISSED** [review]

**Comment by efefury** [comment]
:pray:

**Pull request closed without merging**